gcp/README.md

```diff
@@ -79,3 +81,74 @@ When you're done:
 2. Can you easily obtain the GCP IAM role of the Node?
 3. Can you get the secrets in the SSM Parameter Store and Secret Manager easily? Which paths do you see?
 4. You should see at the configuration details of the cluster that `databaseEncryption` is `DECRYPTED` (`gcloud container clusters describe wrongsecrets-exercise-cluster --region europe-west4`). What does that mean?
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
maybe good to document this a bit better? (E.g. so that public doc is clear why i tis structured like this

Reply:
Added a line above